### PR TITLE
Wire up pending blockchain transactions dropdown

### LINKED ISSUE #251: Wire up pending blockchain transactions dropdown
After the completion of https://github.com/OriginProtocol/demo-dapp/issues/250, wire this up to real transactions.

### PR DESCRIPTION
**Pull request by micahalcorn**
### Checklist:

- [x] Test your work and double check you didn't break anything

### Description:

This resolves #251 🍥 and depends on https://github.com/OriginProtocol/origin-js/pull/285 and...

**does**

- create a `transactions` collection in the redux store
- remove fake data used in prior simulation
- establishe `12` as the necessary block confirmation count for "completion"
- track blockchain transaction confirmations for the following functions:
```
listings.create
listings.close
listings.buy
purchases.buyerConfirmReceipt
purchases.sellerConfirmShipped
purchases.sellerGetPayout
```
- prepare new strings for translation
- move styling that is dependent on "block confirmation count" out of CSS and into the component (inline, for easily switching from `12` to some other number)
- create a utility for mapping custom "transaction types" to contract events (necessary for knowing what `returnValues` to care about in the receipts

**does not**

- track user/identity-related blockchain transaction confirmations

### CODE REVIEW

File: src/components/listing-create.js [hunk 1]
```diff
@@ -135,12 +139,15 @@ class ListingCreate extends Component {
 
   async onSubmitListing(formListing, selectedSchemaType) {
     try {
-      console.log(formListing)
       this.setState({ step: this.STEP.METAMASK })
-      const transactionReceipt = await origin.listings.create(formListing.formData, selectedSchemaType)
+      console.log(formListing)
```
Comment (tyleryasaka) [inline src/components/listing-create.js]: Is this supposed to be here?

Comment (micahalcorn) [reply src/components/listing-create.js]: I just moved it, from above the state change, I guess. 🤷‍♂️

Comment (tyleryasaka) [reply src/components/listing-create.js]: ok cool! I don't like console.log statements to be in source code but that's a battle I'll fight another day haha.

Comment (micahalcorn) [reply src/components/listing-create.js]: Uh, right, so don't look at the messaging-orbit branch 🙈

Comment (tyleryasaka) [reply src/components/listing-create.js]: uh oh

**Pull request merged**